cli.js

```diff
@@ -49,6 +49,7 @@ var cli = meow(
     '  -l, --html   treat input as html (not markdown)',
     '      --mdx    treat input as mdx (not markdown)',
     '  -d, --diff   ignore unchanged lines (affects Travis only)',
+    '  -r, --reporter=REPORTER use a custom vfile-reporter',
```

Comment:
```suggestion
      '     --report <reporter> use custom vfile reporter',
```

Maybe leave the `r` shortcut out? I think it’s fine with just the long form.

And: can you align the options again?